docs/user-guide/patterns/partial-apply.md

```diff
@@ -1,25 +1,51 @@
 (partially parametric rules)=
-Partially Parametric Rules
-==========================
+# Partially Parametric Rules
 
-````{todo}
-
-Write a function that applies a method with some arbitrary arguments and others
-passed in, and uses method otherwise.
+The provided code snippet illustrates a partially parametric rule in CVL that defines specific behavior based on the method invoked (`f`) and its arguments (`calldataargs`). Let's break down the example for better understanding:
 
 ```cvl
-function applyToUser(method f, env e, address user) {
-    if (f.selector == balanceOf(address).selector) {
-        return balanceOf(e, user);
-    }
-    if (f.selector == transfer(address,address).selector) {
-        address other;
-        return transfer(e, user, other);
-    }
-    else {
-        calldataarg arg;
-        return f(e, arg);
-    }
+rule partially_parametric_rule(env e, method f, calldataargs args)
+{
+	if (f.selector == withdraw(uint256, address).selector) {
+		uint256 shares;
+        address to;
```

Comment:
Change the tabs to spaces to keep indentation consistent. Also fix the indentation below.

Reply:
Done

Reply:
still looks off to me in the github diff

Reply:
![image](https://github.com/Certora/Documentation/assets/122722245/7db6f5ee-d2d3-4069-97ae-4ea65ee213aa)

Reply:
![image](https://github.com/Certora/Documentation/assets/122722245/be4b0ce3-c37f-4301-a908-ffbe76dfc4df)